src/ert/config/gen_kw_config.py

```diff
@@ -186,7 +186,7 @@ def from_config_list(cls, gen_kw: list[str | dict[str, str]]) -> list[Self]:
                         params[0], params[1], params[2:]
                     ),
                     forward_init=False,
-                    update=update_parameter,
+                    update="CONST" not in params and update_parameter,
```

Comment:
It would nice to have a warning message saying that when `CONST` distribution the update is set to False. That is in the case that it was changed.